content/pt-br/docs/concepts/policy/resource-quotas.md

```diff
@@ -0,0 +1,649 @@
+
+- Em um cluster com capacidade de 32 GiB de RAM e 16 núcleos, deixe a equipe A usar 20 GiB e 10 núcleos, deixe B usar 10GiB e 4 núcleos e mantenha 2GiB e 2 núcleos em reserva para alocação futura.
+- Limite o _namespace_ "testing" para usar 1 núcleo e 1GiB de RAM. Deixe o namespace "produção" usar qualquer quantia.
+
+Caso a capacidade total do cluster seja menor que a soma das cotas dos _namespaces_, pode haver contenção de recursos. Isso é tratado por ordem de chegada.
+
+Nem a contenção nem as alterações na cota afetarão os recursos já criados.
+
+## Ativando a cota de recursos
+
+O suporte à cota de recursos é ativado por padrão para muitas distribuições do Kubernetes. Isto é
+ativado quando a flag {{< glossary_tooltip text="API server" term_id="kube-apiserver" >}} `--enable-admission-plugins=` tem `ResourceQuota` como
+um de seus argumentos.
+
+Uma cota de recurso é aplicada em um _namespace_ específico quando há um `ResourceQuota` nesse _namespace_.
+
+## Cota de recursos computacionais
+
+Você pode limitar a soma total de [recursos computacionais](/docs/concepts/configuration/manage-resources-containers/) que pode ser solicitado em um determinado _namespace_.
+
+Os seguintes tipos de recursos são suportados:
+
+| Nome do Recurso | Descrição |
+| --------------------- | ----------------------------------------------------------- |
+| `limits.cpu` | Em todos os pods em um estado não terminal, a soma dos limites de CPU não pode exceder esse valor. |
+| `limits.memory` | Em todos os pods em um estado não terminal, a soma dos limites de memória não pode exceder esse valor.|
+| `requests.cpu` | Em todos os pods em um estado não terminal, a soma das solicitações da CPU não pode exceder esse valor. |
+| `requests.memory` | Em todos os pods em um estado não terminal, a soma das solicitações de memória não pode exceder esse valor. |
+| `hugepages-<size>` | Em todos os pods em um estado não terminal, o número de solicitações de grandes páginas do tamanho especificado não pode exceder esse valor. |
+| `cpu` | O mesmo que `requests.cpu` |
+| `memory` | O mesmo que `requests.memory` |
+
+### Cota de recursos para recursos estendidos
+
+Além dos recursos mencionados acima, na versão 1.10, suporte a cotas para [recursos estendidos](/docs/concepts/configuration/manage-resources-containers/#extended-resources) foi adicionado.
+
+Como o `overcommit` não é permitido para recursos estendidos, não faz sentido especificar tanto `requests` e `limits` para o mesmo recurso estendido em uma cota. Portanto, para recursos estendidos, apenas itens de cota com prefixo `requests.` é permitido por enquanto.
+
+Tome o recurso GPU como exemplo, se o nome do recurso for `nvidia.com/gpu` e você quiser limitar o número total de GPUs solicitadas em um _namespace_ para 4, você pode definir uma cota da seguinte maneira:
+
+* `requests.nvidia.com/gpu: 4`
+
+Veja [Viewing and Setting Quotas](#viewing-and-setting-quotas) para mais informações.
```

Comment:
Hi, "Viewing and Setting Quotas", should be translated?

Reply:
I just changed! I will upload along with the other changes soon. Thanks for the suggestion!!